### PR TITLE
OOP school library: unit tests

### PR DESCRIPTION
**Pull request by brighton0102**
### In this milestone I:

1. Add unit tests for all the methods in all the classes:

- Book
- Person
- Student
- Teacher
- Classroom
- Rental

image

### CODE REVIEW

**Comment by tugsem** [comment]
## Invalid Code Review Request ❌ 

Hi @brighton0102 , 

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

<h1>You have submitted a project, that requires equal commit from your partner. If you are allowed to be solo, please attach an screenshot from the student success</h2>

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

_Invalid Code Review Request **does not count** into the code reviews limit._

File: spec/classroom_spec.rb [hunk 1]
```diff
@@ -0,0 +1,30 @@
+require_relative '../classroom'
+require_relative '../student'
+
+Rspect.describe Classroom do
```
Comment (julie-ify) [inline spec/classroom_spec.rb]: - [ ] Your use of `Rspect` here is preventing the specs from being run. This should be `RSpec` (note the capitals R and S)

- [ ] After fixing the issue above, your tests run but 7 of them are failing. Kindly fix them.

  image

**Review by julie-ify — CHANGES_REQUESTED** [review]
## Hi @brighton0102,

Good job so far!

## Highlights!
- Good code organization and working app ✔️
- Descriptive README ✔️ 

Hello ✋, you have done well reaching this point of the milestone 🤝 . But there are some issues that you still need to work on to go to the next project but you are almost there!

# Status: Required Changes ♻️⛔️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread and tag me `@julie-ify` if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by Ghiftee — APPROVED** [review]
## STATUS: APPROVED ♻️♻️
Hi @brighton0102, 

To highlight 👍🏽 
✔️ All tests pass
✔️ No errors

Your project is complete!  There is nothing else to say other than... it's time to merge it 🚀  
Congratulations! 🎉 



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**